x-pack/test/api_integration/apis/management/index_management/component_templates.ts

```diff
@@ -281,8 +281,19 @@ export default function ({ getService }: FtrProviderContext) {
         expect(body).to.eql({
           statusCode: 404,
           error: 'Not Found',
-          message:
-            '[resource_not_found_exception] component template matching [component_does_not_exist] not found',
+          message: 'component template matching [component_does_not_exist] not found',
```

Comment:
The errors we get now are more consistent with what we get from ES, which I think is a good thing for making our error-handling UX more consistent (see https://github.com/elastic/kibana/issues/84801).